src/Scripting/Core/ScriptOptions.cs

```diff
@@ -54,6 +53,7 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 "System.Reflection.Extensions",
                 "System.Reflection.Primitives",
                 "System.Runtime",
+                "System.Runtime.Extensions",
```

Comment:
Ah, never mind. I think I know what the issue is.

---
In reply to: [110982988](https://github.com/dotnet/roslyn/pull/18471#discussion_r110982988) [](ancestors = 110982988,110977656,110973145,110970347)